tools/Az.Tools.Predictor/Az.Tools.Predictor.Test/ModelFixture.cs

```diff
@@ -52,11 +54,24 @@ public ModelFixture()
             var fileInfo = new FileInfo(currentLocation);
             var directory = fileInfo.DirectoryName;
             var dataDirectory = Path.Join(directory, ModelFixture.DataDirectoryName);
-            var commandsModel = ModelFixture.ReadZipEntry(Path.Join(dataDirectory, ModelFixture.CommandsModelZip), ModelFixture.CommandsModelJson);
-            var predictionsModel = ModelFixture.ReadZipEntry(Path.Join(dataDirectory, ModelFixture.PredictionsModelZip), ModelFixture.PredictionsModelJson);
+            var commandsModelVersions= JsonConvert.DeserializeObject<IDictionary<Version, IList<ModelEntry>>>(ModelFixture.ReadZipEntry(Path.Join(dataDirectory, ModelFixture.CommandsModelZip), ModelFixture.CommandsModelJson));
```

Comment:
I wonder whether we can try to deserialize to `PreditiveCommand`. I think that depends on the deserialization behavior: whether it just ignore the fields that's not in `PredictiveCommand`.

Reply:
The issue here is that the server sends the response back with the "Command" property. However, on the model side this same property is named "Suggestion". Since we can't add two aliases for this property to deserialize to, I broke it out into a separate object.

Alternatively, I suppose we can have the server send the response back as "Suggestion" so it matches the model file so this isn't needed. Thoughts?

Reply:
I see what is the gap here. We can leave it as it is right now. I'm ok with changing the server side too.

Reply:
I will leave it as is for now. In my opinion, it is not too big of an issue since the client does not actually download the model file and attempt to parse it and this is only done in the unit tests.